app/src/main/java/com/example/puzzlepic/ui/main/MainFragment.kt

```diff
@@ -61,11 +61,12 @@ class MainFragment : Fragment() {
 
     private lateinit var viewModel: MainViewModel
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
-                              savedInstanceState: Bundle?): View {
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
         _binding = MainFragmentBinding.inflate(inflater, container, false)
-        val view = binding.root
-        return view
+        return binding.root
     }
```

Comment:
For reformatting... 
1) A caution.  Significant reformats can make future commit diffs difficult, as git has to deal with significant reordering of the logic.
2) The IDE can do this automatically, and you can provide a template for the IDE to use.  Ensure all developers are using this same template, or the one that comes by default with the IDE, and have them reformat before committing.  This ameliorates issue 1, above.